packages/amplication-server/src/core/resourceVersion/resourceVersion.service.ts

```diff
@@ -37,7 +37,10 @@ export class ResourceVersionService {
    * create function creates a new resourceVersion for given resource in the DB
    * @returns the resourceVersion object that return after prisma.resourceVersion.create
    */
-  async create(args: CreateResourceVersionArgs): Promise<ResourceVersion> {
+  async create(
+    args: CreateResourceVersionArgs,
+    userId: string
```

Comment:
since we send the notifications for all users in the workspace and not just the current user, we dont need this parameter. 
we can leave it if you think there is another use for it